### ISSUE TITLE
Adjust the displayed map size, when more tasks are loaded (dialog after opting to select nearby tasks)

### ISSUE THREAD

**Issue by govvin** [osmlab/maproulette3#1778]
![image](https://user-images.githubusercontent.com/5821688/177666675-27c31196-d796-4921-a4e2-28ba706cbdea.png)

When a user is given the option to choose the next task, there's a "Load More Tasks" button that they can click, to show nearby tasks.

When this button is clicked, more tasks are made available to the user, but the map doesn't show them by default. An illogical behavior, since the user already clicked the button to review their choices. The user cannot see the additional tasks unless they zoom out manually.

Perhaps, it's more desirable to dynamically adjust the map to display available tasks nearby when they are requested?

>What the user sees by default, after hitting the Load More Tasks button:
>![image](https://user-images.githubusercontent.com/5821688/177666291-72465e21-91db-4f87-9d2f-6dcbcf8e42b4.png)

> What the user may see when they zoom out. More available tasks. 
>![image](https://user-images.githubusercontent.com/5821688/177667116-ad142d68-52a3-473f-b096-74c3c53719ca.png)

> User may assume everything is  "hard" but clicking for more tasks, there are still a few "regular" tasks that may be less daunting for them.
>![image](https://user-images.githubusercontent.com/5821688/177667204-1188e008-a52b-4fec-913a-11c567146936.png)

Basically, the "load tasks" is superfluous and can be combined with zoom behaviour instead.

**Comment by mvexel** [comment]
Good catch. Assigning to next milestone. Let me get back to you if we have any additional questions.

**Comment by govvin** [comment]
Feedback about this fix.

If you're like me, and is using this the "nearby" feature a lot, the current fix for this ticket requires too many scrolling/zooming in, and it doesn't appear to actually show more tasks than what was shown in the previous version.
